VRDR/IJEMortality.cs

```diff
@@ -1077,21 +1077,46 @@ public string SEX_BYPASS
         [IJEField(15, 191, 9, "Social Security Number", "SSN", 1)]
         public string SSN
         {
+
             get
             {
+                string fhirFieldName = "SSN";
+                string ijeFieldName = "SSN";
+                IJEField info = FieldInfo(ijeFieldName);
                 string ssn = record.SSN;
                 if (!String.IsNullOrWhiteSpace(ssn))
                 {
-                    return ssn.Replace("-", string.Empty);
+                    string formattedSSN = ssn.Replace("-", string.Empty).Replace(" ", string.Empty);
+                    if (formattedSSN.Length != info.Length)
+                    {
+                        validationErrors.Add($"Error: FHIR field {fhirFieldName} contains string '{ssn}' which is not the expected length (without dashes or spaces) for IJE field {ijeFieldName} of length {info.Length}");
+                    }
+                    return Truncate(formattedSSN, info.Length).PadRight(info.Length, ' ');
                 }
                 else
                 {
-                    return "";
+                    return new String(' ', info.Length);
                 }
```

Comment:
So, the above could just be

```
    if (formattedSSN.Length != 9)
    {
        validationErrors.Add($"Error: FHIR field SSN contains string '{ssn}' which is not the expected length (without dashes or spaces) for IJE field SSN of length 9");
    }
    return Truncate(formattedSSN, 9).PadRight(9, ' ');
}
else
{
    return new String(' ', 9);
}
```

Reply:
Could argue that 9 is somewhat of a magic number the way I'm suggesting... happy to discuss